packages/react-components/src/components/MentionPopover.tsx

```diff
@@ -299,6 +300,7 @@ export const _MentionPopover = (props: _MentionPopoverProps): JSX.Element => {
   return (
     <div ref={popoverRef}>
       <Stack
+        data-testid={testIds.mentionSuggestionListContainer}
```

Comment:
We should not have test code populating shipped components

Reply:
> We should not have test code populating shipped components

We need the testid as  query when there's no display value of the component. https://testing-library.com/docs/queries/about

Adding test id to the shipped components should be normal. We have `data-ui-id` everywhere which is used for end to end testing. This data-testid is for unit testing.